about.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>About Page</title>
+    <link type="text/css" rel="stylesheet" href="normalize.css">
+    <link type="text/css" rel="stylesheet" href="styles.css">
+</head>
+
+<!--[if lte IE 9]><body class="ieLegacy"><![endif]-->
+<!--[if !IE]><body><![endif]-->
+
+<body>
+    <header class="container">
+        <nav>
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="portfolio.html">Portfolio</a></li>
+            </ul>
+        </nav>
+
+        <h1>About Me</h1>
+    </header>
+
+    <section class="container aboutMe">
+        <article>
+            <p>
+                I am currently attending Mayden Academy to become a full stack developer using HTML, CSS, Java Script, PHP, SQL and Node.js.
+            </p>
+
+            <p>
+                Other interests are back end development using ASP.net and C#.
+            </p>
+        </article>
+    </section>
+
+    <footer class="container">
+        <div class="contactIcon">
+            <span>Contact me</span>
+            <a href="mailto:o.tahboob2000@gmail.com?Subject=Portfolio%20Query"></a>
+            <img src="assets/contactWhite.png" alt="Email me icon">
+        </div>
+
+        <div class="contactIconSmall">
+            <span>Contact me</span>
```

Comment:
not displaying on page - remove if not using

Reply:
Used in IE9 degradation.